Do-It/Controller/DueDatePickerTableViewController.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  DueDatePickerTableViewController.swift
+//  Do-It
+//
+//  Created by Aaron on 2/21/19.
+//  Copyright © 2019 The Swifter Picker-Uppers. All rights reserved.
+//
+
+import UIKit
+
+protocol DueDatePickerTableViewControllerDelegate: AnyObject {
+    func dueDatePickerTableViewController(_ viewController: DueDatePickerTableViewController,
+                                          didSelectTimeRange range: [DateComponents]?)
+}
+
+class DueDatePickerTableViewController: UITableViewController {
+
+    var timeRangeSelected: [DateComponents]?
```

Comment:
This should only ever contain two elements, which you can enforce by making it a two-tuple rather than an array.